gradle/libs.versions.toml

```diff
@@ -1,12 +1,15 @@
 [versions]
-build-config = "4.2.0"
+alchemist = "28.5.4"
+build-config = "4.1.2"
 coroutines = "1.7.3"
 dokka = "1.9.10"
 kotest = "5.8.0"
 kotlin = "1.9.21"
 
 [libraries]
-arrow = { module = "io.arrow-kt:arrow-core", version = "1.2.1" }
+alchemist-api = { module = "it.unibo.alchemist:alchemist-api", version.ref = "alchemist" }
+alchemist = { module = "it.unibo.alchemist:alchemist", version.ref = "alchemist" }
+arrow = "io.arrow-kt:arrow-core:1.2.1"
```

Comment:
Are we using this library? If not, remove it

Reply:
yes, they are used in the alchemist-incarnation-collektive module

Reply:
I was referring to `arrow`, is it being used?

Reply:
yes, we're using the `Option` from this library